src/cba-list-new/shadow.css

```diff
@@ -202,9 +201,133 @@ li .hasInfo::after
   margin-bottom: 1px;
   background: no-repeat var(--icon-project);
 }
+span.row
+{
+  height: 20px;
+}
+
+#group
+{
+  padding-left: 8px;
+  padding-right: 22px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  height: 21px;
+}
+
+.add-item
+{
+  width: 13px;
+  height: 12px;
+  display: inline-block;
+  background: no-repeat var(--icon-group-btn);
+}
+.add-item:after
+{
+  content: "";
+  width: 8px;
+  height: 8px;
+  background: no-repeat var(--icon-group-btn);
+}
+.add-item:hover
+{
+cursor: pointer;
+}
+.control
+{
+  padding-top: 2px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin-left: 22px;
+  height: 20px;
+}
+
+.subitem-btn
+{
+  margin-top: 3px;
+  margin-right: 8px;
+  opacity: 0;
+  width: 15px;
+  height: 15px;
+  display: inline-block;
+  background: no-repeat var(--icon-subitem-btn);
+}
+.subitem-btn:after
+{
+  content: "";
+  width: 8px;
+  height: 8px;
+  background: no-repeat var(--icon-subitem-btn);
+}
+.control:hover .subitem-btn
+{
+  opacity: 1;
+}
+.control:hover .kebab-btn
+{
+  opacity: 1;
+}
 
+.kebab-btn
+{
+  opacity: 0;
+  width: 8px;
+  height: 12px;
+  background: no-repeat var(--icon-kebab-btn);
+}
+.kebab-btn:after
+{
+  opacity: 0;
+  width: 8px;
+  height: 8px;
+  content: "";
+  background: no-repeat var(--icon-kebab-btn);
+}
+.kebab-btn:hover,
+.subitem-btn:hover
+{
+cursor: pointer;
+}
+.control:active .subitem-btn-kebab
```

Comment:
Why are you using `:active` pseudo selector here?

Reply:
It comes from previous scratch. I've deleted it